clj/src/cljd/build.clj

```diff
@@ -316,7 +316,7 @@
                       (println (str "💀 Flutter sub-process exited with " (.exitValue p)))))
                   (finally
                     (some-> p .destroy)))))
-            compilation-success))))))
+            (or compilation-success (System/exit 1))))))))
```

Comment:
`System/exit` is a side effect. It's probably safer to check the result of the `compile-cli` function at the call sites and act accordingly.

Reply:
It's already side-effecty though with the integration of flutter/watch (never returning)